src/ray/common/ray_config_def.h

```diff
@@ -248,7 +248,7 @@ RAY_CONFIG(int32_t, maximum_profile_table_rows_count, 10 * 1000)
 RAY_CONFIG(uint32_t, object_store_get_max_ids_to_print_in_warning, 20)
 
 /// Number of threads used by rpc server in gcs server.
-RAY_CONFIG(uint32_t, gcs_server_rpc_server_thread_num, 1)
+RAY_CONFIG(uint32_t, gcs_server_rpc_server_thread_num, 8)
```

Comment:
`gcs_server_rpc_server_thread_num` is the number of threads that poll incoming requests from gRPC's completion queue. But since GCS is single-threaded, increasing this thread number won't help. IIRC, the object manager is the only place that need multiple RPC threads.

Reply:
I think it also covers request deserialization which sometimes is the bottleneck when request is big.
I remember I'm told that in Ant the number is not 1 either (hopefully my memory is not wrong 😀).